tidy3d/components/microwave/mode_spec.py

```diff
@@ -64,6 +162,26 @@ class MicrowaveModeSpec(AbstractModeSpec, MicrowaveBaseModel):
         "ignored for the associated mode.",
     )
 
+    terminal_specs: Optional[tuple[TerminalSpec, ...]] = pd.Field(
```

Comment:
is there any restrictions on the length of this tuple, like  `< num_modes`? It would also be good to clarify in description whether each terminal_spec is for a separate mode